src/main/java/Main.java

```diff
@@ -54,7 +54,7 @@ static boolean isValidOutputFormatTrue(String str) {
     public static List<String> run(List<String> inputs){
 
         for (String input : inputs){
-            //UserRequest request = UserRequestConverter.convert(input);
+            UserRequest request = UserRequestConverter.convert(input);
             //manager.process(request);
```

Comment:
```suggestion
            manager.process(request);
```
이 주석도 같이 풀어 주셔도 될 것 같은데요?ㅎㅎ 이 PR까지 머지되면 실행해 볼 수 있게요 :)

Reply:
해당 부분 주석 을 풀면 throw exception을 필수로 해줘야되더라구여 null 문제가 생기는건지.., 그래서 뒤에가 다 검증 완료되면 그때 풀고 맞춰서 수정하려하였습니다.

Reply:
아하... 그건 생각을 못했네요 알겠습니다!